### DISCUSSION TITLE
How i can exclude some child element from "group"?

### DISCUSSION THREAD

**Discussion by anbv23** [tailwindlabs/tailwindcss#12261] [Help]
Good day!

I have a specific situation: I need a hover to a child element of a group not to be considered a hover to the entire group.

Please tell me can I do this? 

Revising the layout doesn't suit me, then my animation breaks. I need the group hover to not react to one child of the group.

**Comment by wongjn** [comment]
You could consider `:hover:not(:has())`: https://play.tailwindcss.com/0bCqFtXt6h. However, there is no support in Firefox[^1] at the time of writing so this would need to be thought of as a progressive enhancement.

[^1]: https://developer.mozilla.org/en-US/docs/Web/CSS/:has#browser_compatibility
